documentation/importing-dependencies/tutorial-ways-of-importing.asciidoc

```diff
@@ -88,7 +102,14 @@ structure is created for the `MainLayout` component regardless of availability o
   }
 }
 ----
++
+** In the examples, `/html/layout.html` is loaded and injected before creating the client-side structure for the `MainLayout` component, regardless of the availability of the `/js/animation.js` script.
 
-[NOTE]
-See corresponding tutorials <<tutorial-include-css#,Including Style Sheets>> and <<tutorial-importing#,Importing html/javascript>>
-about static resource paths and URLs.
+// What about the html vs css load order?
```

Comment:
```suggestion
```
I'd guess the CSS in this case as it is inlined into the host page. Technically probably has no effect as the html import is processed eagerly too. But let's not mention that at this point.